packages/client/katello-host-tools/katello-host-tools.spec

```
@@ -135,7 +136,7 @@ BuildArch:  noarch
 Summary:    Adds Tracer functionality to a client managed by katello-host-tools
 Group:      Development/Languages
 
-Requires: katello-host-tools
+Requires: katello-host-tools = %{version}-%{release}
```

This isn't done on line 105 either, but I think it can be:
```suggestion
Requires: %{name} = %{version}-%{release}
```